webapp/src/app.tsx

```diff
@@ -197,6 +214,14 @@ class ScriptSearch extends data.Component<ISettingsProps, ScriptSearchState> {
                     .done()
             }
         }
+        const installScript = (scr: Cloud.JsonScript) => {
+            if (this.modal) this.modal.hide();
+            if (!this.state.packages) {
+                workspace.installByIdAsync(scr.id)
```

Comment:
does this pop a gear wheel dialog? If it is loaded from the web, it might take a few seconds.